CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -545,12 +545,12 @@ msgstr ""
 
 #. 3.2->settings-schema.json->search-filesystem->tooltip
 #. 2.6->settings-schema.json->search-filesystem->tooltip
-msgid "Allows path entry in the menu search box."
+msgid "Allows path entry into the menu search box."
 msgstr ""
 
 #. 3.2->settings-schema.json->show-shutdown-menu->description
 #. 2.6->settings-schema.json->show-shutdown-menu->description
-msgid "Show quit buttons"
+msgid "Show quit buttons on the sidebar"
```

Comment:
Didn't changed that. It will be in a Tab called "Sidebar". I think that would make it clear. Also there no other quit buttons, this unique?